backend/controllers/attendanceStatistics.js

```diff
@@ -0,0 +1,106 @@
+const moment = require('moment')
+const { merge, switchDateFormat, getCurrentShift, pascalToString } = require('../utils')
+
+const getReasonCountMap = (data, reasons) => {
+  const filterAttendance = reason =>
```

Comment:
should really be called `countAttendancesForReason` instead of `filterAttendance`